ci_framework/roles/libvirt_manager/tasks/virsh_checks.yml

```diff
@@ -21,6 +21,34 @@
     state: started
     enabled: true
 
+- name: Hack in the virtproxyd service
```

Comment:
A hack usually needs some deeper explanations for our future self

Reply:
there is the commit message? I may add a comment (same as I wrote in the commit message)